config/SupportedVehicles.xml

```diff
@@ -96,6 +96,30 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_allPurposeTool/all-purpose-tool.xml">
+            <loadingArea offset="0 0.928 0.819" width="3.436" height="1.602" length="1.704"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
```

Comment:
Same issue, zones should be as close to the loading bed as possible. Your Y offset just needs to come down a bit.

![image](https://user-images.githubusercontent.com/15822357/165357441-85ad8383-60e1-4a36-b4f3-caec28d54336.png)

Reply:
Also, we will want to add ```selectedConfigs="1"``` This trailer has an autoload mode and we most likely only want UA when it is not selected.

Reply:
Looks like this trailer has collpasing sides. Maybe also a 
```<options noLoadingIfUnfolded="true"/>``` (Which sounds backwards, but that is the way its labeled on the trailer)

Reply:
@ddewar 
The autoload included by default with the trailer is the same script as the Arcusin bale collector from base game. It was the only solution Bruno figured out to have autoload working for console players. The caveat is that it is only working for bale and PC Players would like to use with pallets. That's why it would be great to have UA also.
Is there a way to have both autoload ?

Reply:
Yes, we can have both. I was just suggesting we use the selectedconfig option so that UA only works when you select Autoload false in the store. That way the player can either have UA or the default Bale loading depending on how the trailer is configured. 

If there are no conflicts with having both enabled at the same time it's probably alright to leave it out. I just didn't know if it would start doing weird things where one Autoload would grab the bale and the other would also try. Loki might have run into this already. I'll tag him too to see if he has any advice on trailers like this.

Reply:
I'm away from my computer, but check out how Loki did it here in the config for a base game trailer:

https://github.com/loki79uk/FS22_UniversalAutoload/blob/main/config/SupportedVehicles.xml#L4

I think the config you want is 1. I checked it yesterday when I was testing yours.